doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -40,18 +57,24 @@ The following steps are made more difficult if code is changing in the "develop"
 Conduct Performance Testing
 ---------------------------
 
+|dedicated|
+
 See :doc:`/qa/performance-tests` for details.
 
 Conduct Smoke Testing
```

Comment:
```suggestion
Conduct Regression Testing
```

Reply:
@landreev Any objections to renaming this section Regression testing vs smoke testing. For the last 2 releases, we have been doing regression testing during this step.